src/main/java/org/jabref/gui/exporter/ExportToClipboardAction.java

```diff
@@ -34,21 +35,24 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(ExportToClipboardAction.class);
 
     // Only text based exporters can be used
-    private static final List<String> SUPPORTED_FILETYPES = Arrays.asList("txt", "rtf", "rdf", "xml", "html", "htm", "csv", "ris");
+    private static final Set<String> SUPPORTED_FILETYPES = Set.of("txt", "rtf", "rdf", "xml", "html", "htm", "csv", "ris");
```

Comment:
Suggestion: convert this field to `Set<FileType>` then you don't need to compare extensions and it's a bit more readable as well.

Reply:
As I doubted it already, it's not possible to compare, because when you create a new TemplateExporter, the FileExtension is created dynamically using newFileType.
And if I compare the Enums directly I would be comparing apples and oranges.

https://github.com/JabRef/jabref/blob/c25b639d82d551e8c2e527ae76f7c53fc4735d90/src/main/java/org/jabref/logic/util/StandardFileType.java#L54-L60

Reply:
Ok, but that's more a shortcoming of this method than a real problem. 
It should be possible to first lookup if the StandardFileType contains a type with matching extensions (similar to the following):
https://github.com/JabRef/jabref/blob/b5002e041d587ffb997e946f55daa9f8bdb438d4/src/main/java/org/jabref/model/entry/field/InternalField.java#L38-L42
and if not return a `UnknownFileType` that implements a proper equals 
https://github.com/JabRef/jabref/blob/b5002e041d587ffb997e946f55daa9f8bdb438d4/src/main/java/org/jabref/model/entry/field/UnknownField.java#L30-L45

Definitely a bit more complex then your solution, but should be a bit more future-proof.

Reply:
Thanks for the hint, applied a similar approach now for the FileTypes